articles/resources/relnotes.md

```diff
@@ -17,6 +17,12 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version 0.12.20100301
+
+*Release date: October 5th, 2020*
+
```

Comment:
load time?

Reply:
The fix shouldn't impact the overall load time. It's just changing the order in which things happen during initialization.

Reply:
@rmshaffer do you want to point to the relevant PR/issue?

Reply:
Good idea. Opened https://github.com/MicrosoftDocs/quantum-docs-pr/pull/1069 for this, please take a look there.